src/Node/Express/App.purs

```diff
@@ -22,119 +22,118 @@ import Node.Express.Internal.App
 import Node.Express.Handler
 
 
+-- | Monad responsible for application related operations (initial setup mostly).
+data AppM e a = AppM (Application -> Eff e a)
```

Comment:
Shouldn't here be not just `e`, but `(express :: EXPRESS | e)` on the right side?